docs/books/computer-games/computer-battle-games/the_vital_message/the_vital_message.py

```diff
@@ -0,0 +1,46 @@
+# Import 'random' so we can generate random numbers.
+# The 'string' module gives us access to string-related utilities
+#The 'time' module helps us to wait for some time
+
+
+import random
+import string
+import time
+print("VITAL MESSAGE")
+
+#Variable to store the difficulty i.e length of string
+D=0
+
+#Loop until the right difficulty level is enterred
+while D<4 or D>10:
+    print("HOW DIFFICULT? (4-10)")
+    x=int(input())
+    D=x
+
+#Function to clear the screen
```

Comment:
Might want to explain *why* this clears the screen - although it's obvious to you and me, the audience for this is people who are just learning to code.

Reply:
@freakboy3742 thank you for your valuable feedbacks and sorry for such mistakes.